### PR TITLE
Fix memory leak with file upload

### PR DESCRIPTION
**Pull request by parfeon**
fix: fix memory leak with file upload

Fix issue because of which `PNFilesManager` leaked each time when PubNub client created.

### CODE REVIEW

File: Example/PubNub/PNAppDelegate.m [hunk 1]
```diff
@@ -77,7 +77,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 - (void)pubNubInit {
 
     // Initialize PubNub client.
-    self.myConfig = [PNConfiguration configurationWithPublishKey:_pubKey subscribeKey:_subKey];
+    self.myConfig = [PNConfiguration configurationWithPublishKey:_pubKey subscribeKey:_subKey uuid:@"pubnub"];
```
Comment (CraigLn) [inline Example/PubNub/PNAppDelegate.m:80]: Was including "pubnub" as a default `uuid` intentional?

**Review by CraigLn — APPROVED** [review]

**Comment by parfeon** [comment]
@client-engineering-bot release

File: Example/PubNub/PNAppDelegate.m (hunk 1 above)
Comment (parfeon) [reply Example/PubNub/PNAppDelegate.m:80]: @CraigLn  `uuid` is **required** field. Later on those values will be changed to random, before client instance allocation.
This example app need really big clean up :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+## v5.1.1
+September 06 2022
+
+#### Fixed
+- Fix issue because of which `PNFilesManager` leaked each time when PubNub client created.
```
Comment (kazydek) [inline CHANGELOG.md:5]:
```suggestion
- Fix the issue because of which `PNFilesManager` leaked each time when PubNub client was created.
```

**Review by kazydek — APPROVED** [review]

**Pull request merged**